packages/govuk-frontend/src/govuk/components/accordion/accordion.puppeteer.test.js

```diff
@@ -760,6 +760,25 @@ describe('/components/accordion', () => {
               'Accordion: Root element (`$module`) is not of type HTMLElement'
           })
         })
+
+        it('throws when the accordion sections are missing', async () => {
+          await expect(
+            renderAndInitialise(page, 'accordion', examples.default, {
+              beforeInitialisation($module, { selector }) {
+                $module
+                  .querySelectorAll(selector)
+                  .forEach((item) => item.remove())
+              },
+              context: {
+                selector: '.govuk-accordion__section'
+              }
+            })
+          ).rejects.toEqual({
+            name: 'ElementError',
+            message:
+              'Accordion: Sections (`.govuk-accordion__section`) not found'
```

Comment:
I'm tempted to suggest we have a new kind of message in `ElementError` of this sort:

`No sections (<IDENTIFIER>) were found`

For when a group of things is empty.

But I wouldn't consider this blocking.

Reply:
Yeah, it does sound better, indeed, when multiple things are missing. Agreed this wouldn't block the PR and can be dealt with separately (wouldn't even block the pre-release, as the content is not part of the public API).